### PR TITLE
adding 'getting started' documentation

### PR DESCRIPTION
**Pull request by sabeegrewal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,26 +10,27 @@ For latest integration of EventHubs and Spark Streaming, the document can be fou
 [Change Log](docs/change_log.md)
 
 ## Usage
+###### For help with Spark Cluster setup and running Spark-EventHubs applications in your cluster, checkout our [Getting Started](docs/getting_started.md) page.
```
Comment (CodingCat) [inline README.md]: we do not need `######` here

File: README.md [hunk 2]
```diff
@@ -10,26 +10,27 @@ For latest integration of EventHubs and Spark Streaming, the document can be fou
...
 
 ### Getting Officially Released Version
 
 We will have the official release in the maven central repo, you can add the following dependency to your project to reference Spark-EventHubs
 
 #### Maven Dependency
-    <!-- https://mvnrepository.com/artifact/com.microsoft.azure/spark-streaming-eventhubs_[2.10 for spark 1.6 or 2.11 for spark 2.x] -->
+    <!-- https://mvnrepository.com/artifact/com.microsoft.azure/spark-streaming-eventhubs_[2.10 for spark 1.6 or 2.11 for spark 2.1.x] -->
```
Comment (CodingCat) [inline README.md]: no, 2.0.x also uses 2.11

File: README.md [hunk 3]
```diff
@@ -10,26 +10,27 @@ For latest integration of EventHubs and Spark Streaming, the document can be fou
...
     <dependency>
         <groupId>com.microsoft.azure</groupId>
-        <artifactId>spark-streaming-eventhubs_[2.10 for spark 1.6 or 2.11 for spark 2.x]</artifactId>
+        <artifactId>spark-streaming-eventhubs_[2.10 for spark 1.6 or 2.11 for spark 2.1.x]</artifactId>
```
Comment (CodingCat) [inline README.md]: same here

File: README.md [hunk 4]
```diff
@@ -10,26 +10,27 @@ For latest integration of EventHubs and Spark Streaming, the document can be fou
...
         <version>[change it to latest version]</version>
     </dependency>
 
 #### SBT Dependency
     // https://mvnrepository.com/artifact/com.microsoft.azure/spark-streaming-eventhubs_2.11
-    libraryDependencies += "com.microsoft.azure" % "spark-streaming-eventhubs_[2.10 for spark 1.6 or 2.11 for spark 2.x]" % "[change it to latest version]"
+    libraryDependencies += "com.microsoft.azure" % "spark-streaming-eventhubs_[2.10 for spark 1.6 or 2.11 for spark 2.1.x]" % "[change it to latest version]"
```
Comment (CodingCat) [inline README.md]: same here

File: README.md [hunk 5]
```diff
@@ -10,26 +10,27 @@ For latest integration of EventHubs and Spark Streaming, the document can be fou
...
 
 #### Maven Central for other dependency co-ordinates
 
-https://mvnrepository.com/artifact/com.microsoft.azure/spark-streaming-eventhubs_[2.10 for spark 1.6 or 2.11 for spark 2.x]/[change it to latest version]
+https://mvnrepository.com/artifact/com.microsoft.azure/spark-streaming-eventhubs_[2.10 for spark 1.6 or 2.11 for spark 2.1.x]/[change it to latest version]
```
Comment (CodingCat) [inline README.md]: same here

File: README.md [hunk 6]
```diff
@@ -52,7 +53,7 @@ You can then add the following dependency to your project to take the pre-releas
     <!-- https://mvnrepository.com/artifact/com.microsoft.azure/spark-streaming-eventhubs_2.11 -->
     <dependency>
         <groupId>com.microsoft.azure</groupId>
-        <artifactId>spark-streaming-eventhubs_[2.10 for spark 1.6 or 2.11 for spark 2.x]</artifactId>
```
Comment (CodingCat) [inline README.md]: same here

File: README.md [hunk 7]
```diff
@@ -71,7 +72,7 @@ In order to build and run the examples, you need to have:
 ## Build Command
     mvn clean
     mvn package
-This command builds and installs EventHubsReceiver jar to local maven cache. Subsequently you can build any Spark Streaming application that references this jar.
+This command builds and installs spark-streaming-eventhubs jar to local maven cache. Subsequently you can build any Spark Streaming application that references this jar.
```
Comment (CodingCat) [inline README.md]: shouldn't we use mvn install here?

**Comment by CodingCat** [comment]
LGTM, and left some comments,

**Comment by sabeegrewal** [comment]
Awesome, thanks!! I'll update everything soon.

**Comment by CodingCat** [comment]
it looks like we have flaky test here...we can look at it later

there is one more comment to fix, can you do that and then I can merge it?

**Pull request merged**